src/main/java/LottoGame.java

```diff
@@ -0,0 +1,50 @@
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+public class LottoGame {
+
+    private final Cost cost;
+    private final Lottos lottos;
+
+    public LottoGame(int cost, NumberGenerator numberGenerator) {
+        this.cost = new Cost(cost);
+        int amounts = calculateLottoAmount();
+        lottos = new Lottos(amounts, numberGenerator);
+    }
```

Comment:
생성자를 통해서 cost를 할당함과 동시에 `Lottos` 객체를 생성하고 있네요. 🤔
생성자는 말그대로 객체를 생성하는 역할만 수행하도록 해보면 어떨까요? 👀

조금 더 구체적으로 이야기해보자면 
외부에서 `Lottos` 객체를 생성해서 생성자 파라미터로 주입받도록 변경해보면 좋을 것 같아요. 😉

```suggestion
    private final Lottos lottos;

    public LottoGame(Lottos lottos) {
        this.lottos = lottos;
    }
```

Reply:
LottoGame이 Lottos만 주입받도록 변경하였습니다. [da30c96](https://github.com/next-step/java-lotto-kakao/pull/12/commits/da30c9687f0529b489042564b35f97d8159a6122)